trunk/lib/PostGalleryImageList.php

```diff
@@ -355,4 +413,27 @@ public static function getFirstImage( $size = 'post-thumbnail', $post_id = null
             'size' => $size,
         ];
     }
+
+
+    public static function getAllAttachmentIds() {
```

Comment:
According to its behavior this is not a getter method. The method should either return all attachment ids or be renamed.
PhpDocs should be added,

Reply:
Renamed to loadAllAttachmentIds